taker-frontend/src/App.tsx

```diff
@@ -182,7 +182,7 @@ export const App = () => {
         />
 
         {closedPositions.length > 0
-            && <Accordion allowToggle width={"100%"}>
+            && <Accordion allowToggle width={"80%"}>
```

Comment:
I am not sure I can follow why this is better, but I don't mind much.

Reply:
This is just a small UI change making the accordion not using 100%.